SUMMARY.md

```diff
@@ -811,7 +811,7 @@
         * [Use MatcherAssert#assertThat(..)](reference/recipes/java/testing/junit5/usehamcrestassertthat.md)
         * [Use Mockito JUnit Jupiter extension](reference/recipes/java/testing/junit5/usemockitoextension.md)
         * [Use OkHttp 3 MockWebServer for JUnit 5](reference/recipes/java/testing/junit5/upgradeokhttpmockwebserver.md)
-        * [Use Vertx JUnit 5 Extension](reference/recipes/java/testing/junit5/vertxunittovertxjunit5.md)
+        * [Use Vert.x JUnit 5 Extension](reference/recipes/java/testing/junit5/vertxunittovertxjunit5.md)
```

Comment:
Not a big deal - but as these are referencing the `reference/recipes` files, this will just get clobbered every release (as this part of the `SUMMARY.md` is overwritten). Would be helpful to leave check spelling on for the rest of the file, though.

Reply:
We can [fix this at the source](https://github.com/openrewrite/rewrite-testing-frameworks/blob/ee9a439948dea63fb3d117f1f0bac39d80e50670/src/main/resources/META-INF/rewrite/junit5.yml#L211-L212) indeed; good that you noticed!